test/HelmetDeclarativeTest.js

```diff
@@ -809,6 +811,32 @@ describe("Helmet - Declarative API", () => {
                 });
             });
 
+            it("use style object in body.", (done) => {
+                /* eslint-disable */
```

Comment:
Why do we need to disable every eslint rule for this block? Worst case scenario could you specify which rules need to be disabled? (similar to line 3 in this file)